src/components/InvertedFlatList/CellRendererComponent.tsx

```diff
@@ -1,20 +1,12 @@
-import PropTypes from 'prop-types';
 import React from 'react';
-import {View} from 'react-native';
+import {StyleProp, View, ViewProps} from 'react-native';
 
-const propTypes = {
-    /** Position index of the list item in a list view */
-    index: PropTypes.number.isRequired,
-
-    /** Styles that are passed to the component */
-    style: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.object), PropTypes.object]),
-};
-
-const defaultProps = {
-    style: {},
+type CellRendererComponentProps = ViewProps & {
+    index: number;
+    style: StyleProp<ViewProps>;
```

Comment:
```suggestion
    style?: StyleProp<ViewProps>;
```

According to the propTypes.